packages/perennial/contracts/multiinvoker/MultiinvokerRollup.sol

```diff
@@ -0,0 +1,290 @@
+pragma solidity ^0.8.0;
```

Comment:
peg to a specific version since this is not a library contract (`0.8.17` probably?)